packages/graphback-datasync/tests/DataSyncConflictProviderTest.ts

```diff
@@ -23,6 +23,10 @@ describe('DataSyncConflictMongoDBDataProvider', () => {
   scalar GraphbackObjectID
   `;
 
+  test('MAX_RETRIES always has a valid value', async () => {
```

Comment:
I think if you move this to outside the `describe` method it should work. the describe method creates a context which this test does not use, so tests are failing